configure.ac

```diff
@@ -452,6 +456,7 @@ AC_ARG_WITH([lz4],
 if test "x$with_lz4" != "xno"; then
   AC_CHECK_HEADERS([lz4.h lz4hc.h])
   AC_CHECK_LIB(lz4,LZ4_decompress_safe)
+  LIBSREQUIRED="$LIBSREQUIRED${LIBSREQUIRED:+ }liblz4"
```

Comment:
@tobim found this is missing an extra check https://github.com/NixOS/nixpkgs/pull/326156#issuecomment-2221360157

Reply:
```suggestion
  if test "x$ac_cv_lib_lz4_LZ4_decompress_safe" = xyes; then
    LIBSREQUIRED="$LIBSREQUIRED${LIBSREQUIRED:+ }liblz4"
  fi
```

Reply:
`libacl` is also needed